starter_code/views/albums.hbs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
you do not need the entire HTML outline in every hbs file. Only in the layout.hbs where you import the {{{body}}}. After that, you can write all other hbs files without the entire layout of the HTML.